stability-checker/deploy/chart/stability-checker/templates/role.yaml

```diff
@@ -13,12 +13,6 @@ rules:
   resources: ["configmaps"]
   verbs: ["create", "get", "list"]
 
-- apiGroups: [""]
```

Comment:
Why are these not necessary anymore?

Reply:
They were used in the same purpose - to add testing addons when we did it using a config map. 
Now we are using dedicated addonsconfigurations CRs, so we don't need that roles anymore.